packages/typespec-ts/test/modularIntegration/usage.spec.ts

```diff
@@ -0,0 +1,93 @@
+import { UsageClient } from "../modularIntegration/generated/models/usage/generated/src/index.js";
+import { assert } from "chai";
+import {
+  UsageContext,
+  createUsage,
+  input,
+  inputAndOutput,
+  output
+} from "./generated/models/usage/generated/src/api/index.js";
+const EXPECTED_VALUE = "example-value";
+describe("UsageContext Classical Client", () => {
```

Comment:
kind of feel like it's not necessary to test every api works both at classical client layer and the api layer in the future. Cadl ranch won't calculate them separately.

Reply:
Yeah I think so, we could only test for classical layer only in future considering the classical would call the api layer.

Reply:
I am okay with it.